drivers/Makefile.dep

```diff
@@ -70,6 +70,13 @@ ifneq (,$(filter bh1750fvi,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter bme680,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_gpio_irq
+  FEATURES_REQUIRED += periph_gpio
+  FEATURES_REQUIRED += periph_i2c
```

Comment:
That means that the module is not working for a board that has SPI but not I2C even though the sensor driver supports both. Maybe it could be solved with
```suggestion
  FEATURES_OPTIONAL += periph_i2c
  FEATURES_OPTIONAL += periph_spi 
```
or something similar.

Reply:
Done.